src/fc.css

```diff
@@ -66,7 +66,7 @@ text {
 .x-axis.top .label,
 .y-axis.left .label,
 .title .label {
-  dominant-baseline: hanging;
```

Comment:
well spotted - I forgot that i had made that tweak. I might move that into a separate PR as it will be buried within these changes.